docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,619 @@
+  - Edge: from 14 to **91**. The minimum version based on Chromium.
+  - Firefox: from 52 to **78**.
+  - Chrome: from 49 to **90**. We have [assumed](https://developers.google.com/search/blog/2019/05/the-new-evergreen-googlebot) that Googlebot is always using the latest version of Chrome.
+  - Safari: from 10 to **12.5**
+
+These changes have allowed us to save [6 kB gzipped](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216) on the `@mui/material` package.
+
+## Design kits
+
+## What's next?
+
+- Start covering the growth of v4, and its inertia, https://material-ui.com/blog/2020-q3-update/
+- These efforts are sustainable, echos back to https://material-ui.com/blog/material-ui-v4-is-out/#premium-themes-store-%E2%9C%A8
+
+### A public roadmap
+
+It's new, link core and x roadmaps.
+
+### Core
+
+#### Unstyled components and hooks
+
+We still have a lot of work to do to have a great set of unstyled components.
+You can keep track of our progress in [#27170](https://github.com/mui-org/material-ui/issues/27170).
+
+Our high-level plan is to use the unstyled components and hooks as the basis of the Material components and second design system.
+We are aiming to complete this work with the next major release (v6).
+As of now, you can evaluate the unstyled primitives in the `@mui/base` package, or check out the next implementation of the Material Design components in the `@mui/material-next` package (targeted at v6).
+Please note that both packages are in an alpha state so that we can release breaking changes –
+we want to take the opportunity to create the best APIs we possibly can.
+
+You can help us shape these new packages by taking part in discussions.
+There are [RFCs](https://github.com/mui-org/material-ui/issues?q=is%3Aopen+label%3Adiscussion+%5BRFC%5D) waiting for responses.
+Don't hesitate to let us know what you think!
+
+#### Joy
+
+### Second design system
+
+We have started the [discussion](https://github.com/mui-org/material-ui/discussions/27803) about the direction of this product which we aim to grow beyond material design.
+
+This new design system, codename `Joy`, will be built on top of our strong foundation packages (`unstyled` and `system`) to provide better DX eg. built-in CSS variables generated from theme, perfect dark mode API, etc.
+
+The plan is to release a set of components in each milestone (ordered by most browse) until it has the same amount of components as `@mui/material`, so that the developers only have to pick a package between material-design and Joy based on the design they want to get started with. Most functionalities and APIs of components will be the same because both design systems are built on top of `@mui/core`. The differences will be the theme structure that we want to achieve the right amount of flexibility and great design out of the box.
+
+We will keep you posted about the progress, stay tuned.
```

Comment:
```suggestion
Developing a new design system is one of the main initiatives, alongside this new brand and the unstyled components, to grow MUI beyond Material Design. Its codename is **Joy** and we have just started [the high-level discussion](https://github.com/mui-org/material-ui/discussions/27803) around it if you want more context. 

It will be built on top of our strong foundation packages (the unstyled components and `system`) to provide a better DX and a different design language. We plan on having built-in CSS variables generated from the theme, a great dark mode API, a custom look and feel, and more.

We're prioritizing developing first the most-searched-for components as we plan on releasing them continuously with each milestone. We intend on having on Joy all the components already available on Material Design so that developers can then pick between them focusing on the design they like the most. The main difference will be at the theme structure, where we want to achieve the right amount of flexibility while providing an amazing design out of the box.

We'll keep you posted about the progress, stay tuned.
```

An overall review of the Joy section :)